Source/Core/Core/DSP/DSPEmitter.h

```diff
@@ -263,6 +264,8 @@ class DSPEmitter : public Gen::X64CodeBlock
   // Counts down.
   // int cycles;
 
+  static std::array<u8, COMPILED_CODE_SIZE> code_region;
```

Comment:
Why would this need to be static? Hypothetically, if you ran two DSPEmitter instances at once, you'd want them to have their own code regions, no?.